.circleci/config.yml

```diff
@@ -192,16 +195,34 @@ jobs:
       - image: circleci/golang:1.9
     <<: *sputnik_vm_steps
 
-  mac_unit_tests:
+  mac_evm_unit_tests:
     macos:
       xcode: "9.0"
+    environment:
+        USE_SPUTNIK_VM: false
     <<: *unit_tests_steps
 
-  linux_unit_tests:
+  linux_evm_unit_tests:
     docker:
       - image: circleci/golang:1.9
-        environment:
-            GOMAXPROCS: 2
+    environment:
+        GOMAXPROCS: 2
```

Comment:
Just out of curiosity, is there is a reason this is the only place `GOMAXPROCS` is set?

Reply:
>commit d08d911cf83f24cef8d7c110ad348f68e9dbc615
>Author: Tomasz Zdybał <tomek@zdybal.lap.pl>
>Date:   Fri Feb 2 01:55:47 2018 +0100
>
>    problem: OOM during unit tests
>    
>    solution: reduce parallelism of tests; set GOMAXPROCS to the number of
>    guaranteed CPU cores